locales/en-US/index.json

```diff
@@ -254,6 +254,45 @@
   "collection.title": {
     "message": "{name} - Collection"
   },
+  "component.modal.modal-creation.header": {
+    "message": "Create a project"
+  },
+  "component.modal.modal-creation.description": {
+    "message": "New projects are created as drafts and can be found under your profile page."
+  },
+  "component.modal.modal-creation.name.label": {
+    "message": "Name"
+  },
+  "component.modal.modal-creation.name.placeholder": {
+    "message": "Enter project name..."
+  },
+  "component.modal.modal-creation.slug.label": {
+    "message": "URL"
+  },
+  "component.modal.modal-creation.visibility.label": {
+    "message": "Visibility"
+  },
+  "component.modal.modal-creation.visibility.description": {
+    "message": "The visibility of your project after it has been approved."
+  },
+  "component.modal.modal-creation.visibility.placeholder": {
+    "message": "Choose visibility.."
```

Comment:
I think it is missing a dot.

Reply:
I don't think so.
there were only two dots.

Reply:
Other placeholders all use three dots.

Reply:
aright，i will fix it